scripts/barclamp_install.rb

```diff
@@ -58,19 +55,41 @@ def usage()
 FileUtils.mkdir log_path unless File.directory? log_path
 log = File.join log_path, "component_install.log"
 
+install_lib_dir = "/var/lib/crowbar/install"
+installation_steps = File.join(install_lib_dir, "installation_steps")
+if File.exist? installation_steps
```

Comment:
Won't work unless I switch the code to Pathname.
(same for all your other comments)

Shall I do that?